### ISSUE TITLE
content cannot be displayed

### ISSUE THREAD

**Issue by ghost** [hugocarreira/jekyll-vision-casper-theme#5]
I like this themes very much,but I have some problems。
I hope you can **help me** with the problem.thank you !!

Slow network is detected. Fallback font will be used while loading.

![qq 20170920141708](https://user-images.githubusercontent.com/28300088/30629449-71c05556-9e0e-11e7-8154-c9493dda1900.png)

**Comment by hugocarreira** [comment]
Yeah, i have a problem with jquery and no have time for fix this..

:(

i accept PR xD

**Comment by ghost** [comment]
. thank you for replying me. hope you solve the problem ，if you have free time.

**Comment by hugocarreira** [comment]
I'll try to give a gas and finish..

**Comment by ghost** [comment]
🆗，thank you very much
